vizro-core/docs/pages/user_guides/dashboard.md

```diff
@@ -184,4 +185,66 @@ To create a dashboard, do the following steps:
             Vizro().build(dashboard).run()
         ```
 
+
+## Adding a dashboard title
+
+By providing a `title` to the [`Dashboard`][vizro.models.Dashboard], a title will be added to each [`Page`][vizro.models.Page]
+on the top of the left navigation panel.
+
+!!! example "Adding a dashboard title"
+    === "app.py"
+        ```py
+        import vizro.plotly.express as px
+        from vizro import Vizro
+        import vizro.models as vm
+
+        df = px.data.iris()
+
+        page = vm.Page(
+            title="My first dashboard",
+            components=[
+                vm.Graph(id="scatter_chart", figure=px.scatter(df, x="sepal_length", y="petal_width", color="species")),
+                vm.Graph(id="bar_chart", figure=px.bar(df, x="sepal_width", y="petal_length", color="species")),
+            ],
+            controls=[
+                vm.Filter(column="species"),
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page], title="Vizro")
+        Vizro().build(dashboard).run()
+        ```
+    === "dashboard.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: scatter
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: scatter_chart
+                type: graph
+              - figure:
+                  _target_: bar
+                  data_frame: iris
+                  x: sepal_width
+                  y: petal_length
+                  color: species
+                id: bar_chart
+                type: graph
+            controls:
+              - column: species
+                type: filter
+            title: My first dashboard
+        title: Vizro
+        ```
+    === "Result"
+        [![DashboardTitle]][DashboardTitle]
+
+    [DashboardTitle]: ../../assets/user_guides/dashboard/dashboard_title.png
+
 After running the dashboard, you can access the dashboard via `localhost:8050`.
```

Comment:
Would add this line under the top example

Reply:
I added it in here, as the linters need some text at the end of the md file. Otherwise they keep running the linters 😄